### ISSUE TITLE
check_vmware_datastore_space plugin Service Ouput contains invalid `MISSING` text

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-vmware#725]
While reviewing some service checks yesterday I noticed that the one-line summary (aka, `Service Output`) contained unexpected `MISSING` text patterns interspersed with normal output.

Example:

> `OK: Datastore HUSVM-DC1-vol6 space usage (0 VMs) is 0.01%!o(MISSING)f 18.0TB with 18.0TB remaining [WARNING: 90%!,(MISSING) CRITICAL: 95%!](MISSING)`

The output was expected to be:

> `OK: Datastore HUSVM-DC1-vol6 space usage (0 VMs) is 0.01% of 18.0TB with 18.0TB remaining [WARNING: 90% , CRITICAL: 95%]`

I reverted from v0.30.5 to the v0.30.4 release and the problem remained.

**Comment by atc0005** [comment]
I tried v0.30.3 and had the same issue. I tried v0.30.2 and the issue went away.

The v0.30.3 release included updates for several dependencies, including an update from v0.8.2 to v0.9.1 of the atc0005/go-nagios library.

After comparing the changes between those two releases, I found the issue:

```go
// ##################################################################
// Note: fmt.Println() (and fmt.Fprintln()) has the same issue as `\n`:
// Nagios seems to interpret them literally instead of emitting an actual
// newline. We work around that by using fmt.Fprintf() and fmt.Fprint()
// for output that is intended for display within the Nagios web UI.
// ##################################################################

// One-line output used as the summary or short explanation for the
// specific Nagios state that we are returning. We apply no formatting
// changes to this content, simply emit it as-is. This helps avoid
// potential issues with literal characters being interpreted as
// formatting verbs.

// Previous (working):
// fmt.Print(es.ServiceOutput)

// Broken:
// fmt.Fprintf(&output, es.ServiceOutput)

// Now (not merged), working:
fmt.Fprint(&output, es.ServiceOutput)
```

Evidently when applying sweeping bulk changes to replace `fmt.Print*` variants with `fmt.Fprint*` equivalents I unintentionally swapped out the function that prints content as-is for the function that looks for formatting patterns (aka, "verbs") and attempts to perform interpolation using supplied values. In this case, there were no supplied values, hence the `MISSING` text spread throughout the output.

I generated a new local build of the plugin for testing and confirmed that the problem was resolved.

This will need to be fixed in the upstream library project and included here as part of a new stable (v0.30.6) release.

**Comment by atc0005** [comment]
See also:

- https://github.com/atc0005/go-nagios/issues/139
- https://github.com/atc0005/go-nagios/issues/58

**Comment by atc0005** [comment]
> This will need to be fixed in the upstream library project and included here as part of a new stable (v0.30.6) release.

Fixed in the upstream v0.9.2 release.
